include/ada/implementation.h

```diff
@@ -29,21 +29,13 @@ namespace ada {
    *
    * @param input the string input to analyze.
    * @param base_url the optional string input to use as a base url.
-   * @param encoding encoding (default to UTF-8)
```

Comment:
We may still want to document that we expect UTF-8. It is better to be explicit.

Reply:
Just updated the description. Nice catch!